File: edukate-notifier/src/main/java/io/github/sanyavertolet/edukate/notifier/EdukateNotifierApplication.java
package io.github.sanyavertolet.edukate.notifier;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {
        "io.github.sanyavertolet.edukate.common.configs",
        "io.github.sanyavertolet.edukate.common.security",
        "io.github.sanyavertolet.edukate.notifier"
})
public class EdukateNotifierApplication {
    public static void main(String[] args) {
        SpringApplication.run(EdukateNotifierApplication.class, args);
    }
}

File: edukate-notifier/src/main/kotlin/io/github/sanyavertolet/edukate/notifier/EdukateNotifierApplication.kt
package io.github.sanyavertolet.edukate.notifier

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication(
    scanBasePackages =
        [
            "io.github.sanyavertolet.edukate.common.configs",
            "io.github.sanyavertolet.edukate.common.security",
            "io.github.sanyavertolet.edukate.notifier",
        ]
)
@ConfigurationPropertiesScan(basePackages = ["io.github.sanyavertolet.edukate.notifier.configs"])
class EdukateNotifierApplication

@Suppress("SpreadOperator")
fun main(args: Array<String>) {
    runApplication<EdukateNotifierApplication>(*args)
}
